core/src/main/java/hudson/model/View.java

```diff
@@ -813,29 +831,35 @@ public AsynchPeople(View parent) {
             int itemCount = 0;
             for (Item item : items) {
                 for (Job<?,?> job : item.getAllJobs()) {
-                    if (job instanceof AbstractProject) {
-                        AbstractProject<?,?> p = (AbstractProject) job;
-                        RunList<? extends AbstractBuild<?,?>> builds = p.getBuilds();
+                    if (job instanceof SCMTriggerItem) {
```

Comment:
The only thing I can think of is to avoid iterating builds of `ExternalJob`, but that seems a pretty marginal case.